test/jackdaw/test_test.clj

```diff
@@ -0,0 +1,156 @@
+(ns jackdaw.test-test
+  (:require
+   [clojure.test :refer :all]
+   [jackdaw.serdes.avro.schema-registry :as reg]
+   [jackdaw.streams :as k]
+   [jackdaw.test :as jd.test]
```

Comment:
Elsewhere, we've used the equivalent of `[jackdaw.test :as jt]`. I've also seen '[jackdaw.test :as j.test]`. We should standardize on one of these three.